src/pages/ServiceDetails/ServiceDescription.tsx

```diff
@@ -69,18 +69,20 @@ class ServiceDescription extends React.Component<ServiceInfoDescriptionProps, St
     const workloads: AppWorkload[] = [];
     if (this.props.serviceDetails) {
       if (this.props.serviceDetails.workloads) {
-        this.props.serviceDetails.workloads.forEach(wk => {
-          if (wk.labels) {
-            const appName = wk.labels[serverConfig.istioLabels.appLabelName];
-            if (!apps.includes(appName)) {
-              apps.push(appName);
+        this.props.serviceDetails.workloads
+          .sort((w1: WorkloadOverview, w2: WorkloadOverview) => (w1.name > w2.name ? -1 : 1))
```

Comment:
This logic is flipped, look into the other side:
![image](https://user-images.githubusercontent.com/1662329/122900489-07b24c00-d34d-11eb-8765-b2f0e4c76bb9.png)
(sorry mazz)

Reply:
But it sorts correctly.
![Screenshot from 2021-06-22 17-53-07](https://user-images.githubusercontent.com/604313/122958081-d48aaf80-d382-11eb-8569-c7bf4dc1e549.png)